src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -176,6 +176,22 @@ class BagsServletSpec extends TestSupportFixture
     }
   }
 
+  it should "fail when the given uuid is too long" in {
+    val tooLongUuid = "01000000-0000-0000-0000-0000000000011234465565"
+    get(s"/${ tooLongUuid }") {
+      status shouldBe 400
+      body shouldBe s"invalid UUID string: $tooLongUuid"
+    }
+  }
+
+  it should "fail when the given uuid is too short" in {
```

Comment:
If i am correct the difference between 8 and 9 is the check 

`int len = name.length();
if (len > 36) {
    throw new IllegalArgumentException("UUID string too large");
}`

So i should only check if the string is too long and drop the regex matcher?

Reply:
@rvanheest : the short UUID may be valid, but I think it would be better to require the client to add the leading zeroes. If somehow the string got shortened by one char, we would catch that error.

Reply:
We now validate if the string is exactly 36 characters long